tests/unit/modules/test_pf.py

```diff
@@ -66,14 +66,29 @@ def test_disable_when_disabled(self):
         with patch.dict(pf.__salt__, {'cmd.run_all': mock_cmd}):
             self.assertFalse(pf.disable()['changes'])
 
-    def test_loglevel(self):
+    def test_loglevel_freebsd(self):
```

Comment:
Why are we changing an existing test to apply specifically to FreeBSD? This should be a separate test or this test should be extended to support both the existing and the new instead of modified like this.

Reply:
The existing tests do it both possible ways; I was looking at e.g. `test/unit/modules/test_cron.py`, but there are also tests that have all the different OS variations in one test case (e.g. `./integration/states/test_pkg.py`). I'm not particularly attached to either, but I can definitely rewrite as one TC.